.github/workflows/ci.yaml

```
@@ -36,6 +36,9 @@ jobs:
       - name: Run tests
         run: bazel test --test_env=NO_CLEANUP=1 //...
 
+      - name: Run tests (bzlmod enabled)
```

I think we'll need something like (untested):
```suggestion
      - name: Run tests (bzlmod enabled)
        if: ${{ matrix.bazel-version != '5.x' }}
```
To make CI green